src/Components/QuestionComponentContainer/QuestionComponentContainer.js

```diff
@@ -175,56 +177,72 @@ const QuestionComponentContainer = ({
   };
 
   const renderHeaderAndSubheader = () => {
```

Comment:
Can you pull from main, I made a big change to this function in the last release, that moved most of this logic to `Assets/questions.tsx`. I see that there is a subheader, so you might need to add a subheader in `Assets/questions.tsx`. Let me know if you need help updating it.

Reply:
Merged main in and added subheader for step 3. Subheader Roboto Slab font should show once Marianna's ticket has been merged in. Also, I noticed an error in the console for step 2 due to a <p> element having a div child. This has been fixed. Screenshots below:

![image](https://github.com/Gary-Community-Ventures/benefits-calculator/assets/7133950/c2b63e4c-750d-418f-9d10-d343599898b2)

<img width="1651" alt="Screenshot 2023-12-13 at 11 46 00 AM" src="https://github.com/Gary-Community-Ventures/benefits-calculator/assets/7133950/1ca49aa3-be31-438b-85a0-90ce9c14f4a8">

![image](https://github.com/Gary-Community-Ventures/benefits-calculator/assets/7133950/85319b77-a866-446a-bf66-501889619017)